content/en/guides/core/registry/model_registry_eol.md

```diff
@@ -87,6 +136,22 @@ For more details and to learn about the full range of features in the W&B Regist
 
 ## FAQs
 
+### How much notice will we receive before our organization is scheduled for migration?
+
+W&B is committed to providing sufficient notice. Expect to receive direct communication and see an advisory banner in the W&B UI approximately 4-6 weeks before your organization's scheduled migration window. This will give your team time to prepare and ask any questions. 
```

Comment:
Will defer to @trane293 on this, but looks like we're aiming for a 2 week notice now